src/components/styles/tokens/colors.module.css

```diff
@@ -0,0 +1,3 @@
+@value border-box-shadow: rgba(0, 0, 0, 0.03);
```

Comment:
let's give the `color-` prefix for all colors:

```
color-box-shadow
color-border
color-background-primary
```

as well for the layout in the future:

```
layout-widget-width
layout-padding-vertical
layout-padding-horisontal
...
```